modules/src/mock_client/state.rs

```diff
@@ -74,6 +91,65 @@ impl ClientState for MockClientState {
         // TODO
         false
     }
+
```

Comment:
Let's purge this (or move it in the correct spot, which is probably `mock_client/client_def.rs`).